config/src/config/mempool_config.rs

```diff
@@ -24,6 +25,7 @@ impl Default for MempoolConfig {
             shared_mempool_tick_interval_ms: 50,
             shared_mempool_batch_size: 100,
             shared_mempool_max_concurrent_inbound_syncs: 100,
+            shared_mempool_min_broadcast_recipient_count: 3,
```

Comment:
why 3 is default? how will it work with existing setup where VFN only has 1 upstream?

Reply:
what value do you suggest? The mempool peer manager implementation handles the case where # existing upstream peers < `min_broadcast_recipient_count` - it will just be best effort and ends up broadcasting to all upstream peers possible (in your example, the VFN will try to broadcast to 3 but ends up sending to the 1 upstream peer)